dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/agent/FamilyNameServiceImpl.java

```diff
@@ -1,22 +1,29 @@
 package de.digitalcollections.cudami.server.business.impl.service.identifiable.agent;
 
+import de.digitalcollections.cudami.server.backend.api.repository.identifiable.IdentifierRepository;
 import de.digitalcollections.cudami.server.backend.api.repository.identifiable.agent.FamilyNameRepository;
 import de.digitalcollections.cudami.server.business.api.service.identifiable.agent.FamilyNameService;
+import de.digitalcollections.cudami.server.business.api.service.identifiable.alias.UrlAliasService;
 import de.digitalcollections.cudami.server.business.impl.service.identifiable.IdentifiableServiceImpl;
 import de.digitalcollections.model.identifiable.agent.FamilyName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
+@Transactional
```

Comment:
Muss man hier keine Exception-Klassen mehr angeben?

Reply:
Ich schreib's nochmal explizit hier hin :wink: : `@Transactional` führt ein Rollback nur bei `RuntimeException`s und davon abgeleiteten Exceptions aus. In manchen Klassen werfen wir sog. Checked Exceptions (`CudamiServiceException`, `ValidationException`). Damit diese Exceptions ebenfalls einen Rollback auslösen, muss man `@Transactional(rollbackFor = {...})` angeben. Wenn wir in einer Klasse keine Checked Exceptions werfen (und das ist leider noch in vielen Klassen der Fall), kann man sich den `rollbackFor`-Parameter sparen.

Reply:
Alles klar, danke für die Erklärung :bulb: